packages/mc-scripts/src/utils/get-config-diff.js

```diff
@@ -0,0 +1,269 @@
+const chalk = require('chalk');
+
+// Since not all terminal supports colors, to make things more consistent for testing purposes,
+// during tests the color used is appended before the string instead of coloring it.
+const isTest = process.env.NODE_ENV === 'test';
+const red = (str) => {
+  if (isTest) return 'red-' + str;
```

Comment:
Maybe we can wrap the string like

```suggestion
  if (isTest) return `<color-red>${str}</color-red>`
```